introduction.md

```diff
@@ -30,29 +22,30 @@ Contents:
 
-The Emergency TX is the first to be signed. It would spend a deposit and pay to the `emergency output descriptor` which is a bitcoin script that is harder to unlock than `deposit output descriptor`. Keeping the `emergency output descriptor` private among stakeholders strengthens the emergency deterrent feature as attackers don't know how they would compromise the emergency vault.
+During the set-up, stakeholders prepare an *emergency output descriptor* which is a bitcoin script that is harder to unlock than *deposit output descriptor*. Keeping the *emergency output descriptor* private among stakeholders strengthens the emergency deterrent feature as attackers don't know how the emergency wallet is protected before it is spent from.
 
-Then, stakeholders exchange signatures for the Cancel and Unvault Emergency TXs:
+The stakeholders exchange signatures for the Cancel, Emergency and Unvault Emergency TXs:
 
-- The Unvault Emergency TX would spend the Unvault TX to pay to the `emergency output descriptor`.
-- The Cancel TX would spend the Unvault TX to pay back to the `deposit output descriptor`.
+- The Cancel TX would spend the Unvault TX to pay back to the *[deposit output descriptor](transactions.md#out)*.
+- The Emergency TX would spend a deposit and pay to the *[emergency output descriptor](transactions.md#emergency_txs)*.
+- The Unvault Emergency TX would spend the Unvault TX to pay to the *[emergency output descriptor](transactions.md#emergency_txs)*.
 
-This way, funds are protected _before_ delegation to fund managers. Finally, the Unvault TX is signed. It spends the Deposit TX and pays to either:
+This way, funds are protected _before_ delegation to fund managers. Eventually, the Unvault TX may be signed (to restrictively delegate the UTXO to managers). It spends from the *[deposit output descriptor](transactions.md#out)* and pays to the *[unvault output descriptor](transactions.md#out-1)*, and is accessible with either:
 
-- the managers' K-of-M multi-signature + J cosigning servers + time-lock of X blocks, or 
-- to the stakeholders' N-of-N multi-signature without delay. 
+- the managers' K-of-M multi-signature + time-lock of X blocks + J cosigning servers (where J = 0 if deploying without co-signers or N of deploying with co-signers), or
+- the stakeholders' N-of-N multi-signature (with no delay).
 
-Managers can attempt to use the Spend TX to pay external wallets, but are forced to wait for the Unvault time-lock to expire before their transaction finalizes. During that delay period, a Cancel or Unvault Emergency TX can be broadcasted if the manager is breaching pre-defined policy. 
+Managers broadcast the relevant Unvault TX, and can attempt to use the Spend TX to pay external wallets, but are forced to wait for the Unvault time-lock to expire before their transaction finalizes. During that delay period, a Cancel or Unvault Emergency TX can be broadcasted which will revoke the spend attempt.
```

Comment:
I'm sorry for nit'ing again.. I really did not want to but i find this too misleading.

> Managers broadcast the relevant Unvault TX, and can attempt to use the Spend TX to pay external wallets,, but are forced to wait for the Unvault time-lock to expire before their transaction finalizes.

Is it me or this seems to imply the managers broadcast the Spend right after broadcasting the Unvault?
Also, if we keep it like that i'd at least say "to expire before the Spend transaction becomes valid".